deployment/hasura/migrations/Aerie/3_sequence_parcel/down.sql

```diff
@@ -0,0 +1,203 @@
+
+------------------------------------
+--- Modify Expanded Sequences Table
+------------------------------------
+alter table sequencing.expanded_sequences
+  add column edsl_string text;
+
+update sequencing.expanded_sequences
+set edsl_string = '{}'
+where edsl_string is null;
+
+alter table sequencing.expanded_sequences
+  alter column edsl_string set not null;
+
+------------------------------------------
+---- Modify the Command Dictionary Table
+------------------------------------------
+
+alter table sequencing.command_dictionary
+  add column command_types_typescript_path text;
+
+--- Data Migration
+update sequencing.command_dictionary
+set command_types_typescript_path = dictionary_path
+where command_types_typescript_path is null;
+
+
+alter table sequencing.command_dictionary
+  alter column command_types_typescript_path set not null;
+
+---
+
+drop trigger set_timestamp on sequencing.command_dictionary;
+
+comment on column sequencing.command_dictionary.command_types_typescript_path is e''
+  'The location of command dictionary types (.ts) on the filesystem';
+
+alter table sequencing.command_dictionary
+  drop column updated_at,
+  drop column dictionary_path;
+
+--------------------------------------
+---- Modify the Expansion Rule Table
+--------------------------------------
+alter table sequencing.expansion_rule
+  add column authoring_command_dict_id integer;
+
+comment on column sequencing.expansion_rule.authoring_command_dict_id is e''
+  'The id of the command dictionary to be used for authoring of this expansion.';
+
+--- Data Migration
+update sequencing.expansion_rule
+set authoring_command_dict_id = p.command_dictionary_id
+from sequencing.expansion_rule er
+       inner join sequencing.parcel p on er.parcel_id = p.id;
```

Comment:
This `update` can be reworked to avoid the join:
```sql
update sequencing.expansion_rule er
set authoring_command_dict_id = p.command_dictionary_id
from sequencing.parcel p 
where er.parcel_id = p.id;
```